tests/examples/opinions/united_states/conn_example.compare.json

```diff
@@ -451,13 +451,13 @@
   },
   {
     "case_dates": "2023-06-20",
-    "case_names": "Connex Credit Union v. Thibodeau",
+    "case_names": "SC20737 - Cohen v. Rossi SC20737 First Concurrence - Cohen v. Rossi SC20737 Second Concurrence - Cohen v. RossiSC20630 - Clark v. Waterford, Cohanzie Fire Dept.SC20630 Dissent - Clark v. Waterford, Cohanzie Fire Dept.",
```

Comment:
This is the wrong case name from the comment above

Reply:
this is because connex - case has no dash - but the code should handle this case now.